README.md

```diff
@@ -5,6 +5,9 @@ cluster inside a VM. You can try out OpenShift or develop with it, day-to-day, o
 
 Minishift uses [libmachine](https://github.com/docker/machine/tree/master/libmachine) for
 provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) for running the cluster.
+For general information about OpenShift cluster architecture,  
```

Comment:
Also why "general"? Is "more" not better? What does "general" mean  in this context anyways?